src/components/lyrics.tsx

```diff
@@ -0,0 +1,74 @@
+import { useEffect, useState } from "react";
+
+export default function Lyrics({ lyricsLink }: { lyricsLink: string }) {
+  const [translatedLyrics, setTranslatedLyrics] = useState<string | null>(null);

```

Comment:
you could use one state for both lyrics, just store data in object with fields: lyrics and translated lyrics, also you could fetch translated lyrics immediately after not translated instead of button click, but idk if it is right way